src/components/Compare/CompareMain.tsx

```diff
@@ -249,14 +248,9 @@ const CompareMain = (props: {
     return null;
   }
 
-  // TODO(chris): pusing down use of county as far as possible, but underlying compare
-  // table code not yet ready to stop using county as an input, so querying the legacy Location
-  // county API here.
-  const locationCounty = findCountyByFips(currentCounty?.region.fipsCode || '');
```

Comment:
this analysis was a little trickier.  Ultimately, everywhere this was used was just boolean, which could be ascertained by just passing currentCounty in a boolean context, saving this expensive lookup (which scanned looped through all states and counties!) and in fact, eliminating the only use of this function.